site/content/en/docs/tasks/manage/administer_cluster_quotas.md

```diff
@@ -412,19 +412,11 @@ will present to the Kubernetes Scheduler when the Workload is admitted by
 a ClusterQueue. Customizations are defined by specifying resource transformations
 in the Kueue configuration as a cluster-level setting.
 
-{{< feature-state state="alpha" for_version="v0.9" >}}
```

Comment:
I would revert it and said that it is beta and enabled by default. Like we have in other places.

Reply:
Done